test/xpu_api/utilities/meta/meta.trans/meta.trans.other/common_type.pass.cpp

```diff
@@ -0,0 +1,279 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+// type_traits
+
+// common_type
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+#include <oneapi/dpl/functional>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+#include "support/utils_invoke.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+struct E
+{
+};
+
+template <class T>
+struct X
+{
+    explicit X(T const&) {}
+};
+
+template <class T>
+struct S
+{
+    explicit S(T const&) {}
+};
+
+namespace std
```

Comment:
Should it be specialized in namespace `oneapi::dpl`?

Reply:
Fixed: defined in `std` namespace

Reply:
Fixed